docs/concept/Generators.md

```diff
@@ -121,6 +121,22 @@ A few notes about sizes. iOS size has no upper limit, so be careful with the val
 
 Lastly, the splash-screen generator will exit if your input file has not been modified. The same source equality check, as the one on the app-icon generator, will encourage you to make customizations before using the generator (see the `--skip-source-equality-validation` section above).
 
+## CLI Options
+
+### `--exact`
+
+The default filename is PascalCased based on the name you pass in to the generate command. For example:
+
+`npx ignite-cli@latest g screen Episodes` will generate `EpisodesScreen.tsx` in the case of the default generator template `NAMEScreen.tsx`.
+
+This `--exact` switch specifies the generated filenames (`NAME` in the filename of your template) will be how you pass it in. For example:
+
+`npx ignite-cli@latest g screen log-in` will generate `log-inScreen.tsx`
```

Comment:
We do want `log-in.tsx` and it does do that as well if you name your generator `NAME.tsx.ejs`

I wanted to craft the documentation in such a way that it explained things if you weren't using Expo Router, since Ignite doesn't by default. So this snippet shows what the default screen generator template would do.

We could show two examples though, if you think that is better.

Reply:
IMO, `npx ignite-cli@latest g screen log-in` will not result in `log-inScreen.tsx` but rather `log-in.tsx`. As in, it'll not only avoid the casing issue, but also just straight up use the name as-is and not add Screen to it.

Reply:
@jamonholmgren So if `--case=none` for example (currently `--exact`) you're saying to ignore the template filename `NAMEScreen.tsx` completely?

To be clear, it doesn't do this if you just name the template `NAME.tsx`, it's only adding screen in the example because that is what our default generator is named

### npx ignite-cli g screen --exact log-in
| tpl filename    | generated filename |
| -------- | ------- |
| NAMEScreen.tsx    | log-inScreen.tsx |
| NAME.tsx    | log-in.tsx |

Reply:
Updated, see if this is more clear and if we want it changed still

![image](https://github.com/user-attachments/assets/36f60180-d724-4849-9cb5-ec86c4188209)